src/content/docs/style-guide/structure/styleguide-markup-indentation.mdx

```diff
@@ -0,0 +1,364 @@
+        * Are the policies necessary? Do they have business impact?
+        * Are the thresholds set properly?
+5. Technical recommendations: Here, you and the stakeholders will review any technical recommendations, including:
+    * Are there application / system problems for engineering to review?
+    * Are there poorly constructed policies that need to be fixed?
+    * Are there instrumentation gaps?
+
+## Indentation of consecutive procedures [#consecutive]
+
+As long as consecutive procedures are left-justified, they behave as you might expect in Markdown. The following example shows how you'd set up two left-justified, consecutive lists in Markdown:
+
+<img
+  title="Screenshot showing left-justified consecutive procedures"
+  alt="Screenshot showing left-justified consecutive procedures"
+  src={indentLeftJustified}
+/>
+
+However, when you indent consecutive procedures, such as inside a collapser, Markdown does some unusual formatting:
+
+* Markdown doesn't indent the introduction of the second procedure.
+* Markdown starts the numbering of the second procedure based on the last number of the first procedure.
+
+Here's an example of how the Markdown for same consecutive procedures are indented inside of a collapser. Note how the introduction for the second procedure isn't indented (see "You can also add...") and note how the numbering for the second procedure starts at 5:
+
+<CollapserGroup>
+  <Collapser
+    id="incorrect-formatting"
+    title="Two consecutive lists with incorrect formatting"
+  >
+    When you add tags via the UI or API, this occurs at the entity level. This means that, for example, if you have multiple APM agents monitoring a single entity and apply a tag via the UI, that tag is available for all data coming from that entity.
+
+    To add or delete tags via the UI:
+
+      1. Select a monitored entity, like a monitored app or host.
+      2. Near the entity’s name at the top of the page, select the **See metadata and manage tags** menu.
+      3. In the menu that pops up, add or delete a tag.
+      4. For APM agents, restart your service.
+
+    You can also add or delete tags associated with a NRQL alert condition. To add or delete condition tags via the UI:
+
+      1. Navigate to a condition.
+      2. Near the condition’s name at the top of the form, click **Manage tags**.
+      3. In the menu that pops up, add or delete a tag.
+
+    To manage tags via API: see our [NerdGraph tagging docs](/docs/apis/nerdgraph/examples/nerdgraph-tagging-api-tutorial). For a guide to using our CLI to automate tagging, see [our developer site](https://developer.newrelic.com/automate-workflows/5-mins-tag-resources/).
+  </Collapser>
+</CollapserGroup>
+
+To avoid this unfortunate indentation and numbering, insert a second blank line feed so you have two line feeds separating the introductory sentence of the second procedure from the last numbered item of the first list. Here's now that mark-up looks:
```

Comment:
This information should be in the first sentence of the section, so the solution isn't hidden inside a section. I'm not sure how you want to reorganize the info, but definitely move this up.